addons/point_of_sale/static/tests/tours/helpers/PartnerListTourMethods.js

```diff
@@ -13,3 +13,73 @@ export function clickPartnerDetails(name){
         trigger: `.partner-list tr:contains("${name}") button.edit-partner-button`,
     }
 }
+
+export function checkContactValues(name, address = "", phone = "", mobile = "", email = "") {
+    const steps = [
+        {
+            content: `Check partner "${name}" from partner list screen`,
+            trigger: `.partner-list td:contains("${name}")`,
+            run: () => {},
+        },
+        {
+            content: `Check address "${address}" for partner "${name}"`,
+            trigger: `.partner-list tr:contains("${name}") .partner-line-adress:contains("${address}")`,
+            run: () => {},
+        },
+    ];
+
+    if(phone) {
+        steps.push({
+            content: `Check phone number "${phone}" for partner "${name}"`,
+            trigger: `.partner-list tr:contains("${name}") .partner-line-email:contains("${phone}")`,
+            run: () => {},
+        });
+    }
+
+    if(mobile) {
+        steps.push({
+            content: `Check mobile number "${mobile}" for partner "${name}"`,
+            trigger: `.partner-list tr:contains("${name}") .partner-line-email:contains("${mobile}")`,
+            run: () => {},
+        });
+    }
+
+    if(email) {
+        steps.push({
+            content: `Check email address "${email}" for partner "${name}"`,
+            trigger: `.partner-list tr:contains("${name}") .partner-line-email .email-field:contains("${email}")`,
+            run: () => {},
+        });
+    }
+
+    return steps;
+}
+
+export function searchCustomer(val) {
```

Comment:
Just a detail but just from the name of the function we could think this does the same as the existing function `clickCustomer`. Maybe we could rename it to something like `searchCustomerField` or `searchCustomerValue`.